src/status_im/transport/filters.cljs

```diff
@@ -22,14 +22,14 @@
                        (re-frame/dispatch [:shh.callback/filter-removed chat-id])))))
   (log/debug :stop-watching filter))
 
-(defn add-filters!
+(defn add-raw-filters!
   [web3 filters]
-  (log/debug "PERF" :add-filters (first filters))
+  (log/debug "PERF" :add-raw-filters filters)
   (re-frame/dispatch
    [:shh.callback/filters-added
     (keep
      (fn [{:keys [options callback chat-id]}]
-       (when-let [filter (.newMessageFilter
+       (when-let [filter (.newRawMessageFilter
```

Comment:
seems like the last use of web3 could you use json rpc here so we can finaly get rid of it?

Reply:
this one I need for the polling (I just need the `Filter` js object), it actually does not make any request through RPC, but we can get rid of it all together once we use signals for messages, once we get closer to completion of the PR we can revisit and we can decide whether to include it in this one or a separate, depending on how much stuff there is to test